docs/examples/llama_datasets/downloading_llama_datasets.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>have the same confusion here.</p><p>what is the difference between <strong>reference_contexts, reference_answer, response, contexts</strong></p><p> </p><p>probably need to give more context</p>

Reply:
<p>Oh, the reference contexts, are meant to just represent the chunks of the source documents that are relevant to the reference answer. The doc string for <code>LabelledRagDataExample</code> describes what <code>reference_contexts</code> and <code>reference_answer</code> represent, though this doc string needs to be updated to included <code>query_by</code> and <code>reference_answer_by</code> fields. For <code>response</code> and <code>contexts</code> these can be found in <code>RagExamplePrediction</code> class — I'll update the doc string for it. Essentially these represent the "predicted" versions of the <code>reference_answer</code> and <code>reference_contexts</code> , respectively. I believe I have adopted the similar terminology we've used throughout our <code>evaluation</code> module up to this point.</p><p> </p><p>In any case, I'll make a note on these fields in the <code>labelled-rag-dataset.ipynb</code> rather than here as that one is more about introducing the new abstractions. :) </p><p> </p><p>Good point about labelling what specific "ai" was used. We should just be able to plug in the LLM model name here if an "ai" was used.</p><p> </p><p> </p><p>  </p>

Reply:
`represent the chunks of the source documents that are relevant to the reference answer` -- Was this found using embeddings? If so, we should also include the embeddings used

Reply:
<p>It may not necessarily be found via embeddings. For example, if it was human-generated example, then the human just found the context text and wrote their custom question/reference_answer for it.</p><p> </p><p>Also for our dataset generator, we don't really create an index and do retrieval. (We do build a summary index so that all the nodes are used, though). This really just creates questions per node though... so I don't think any retrieval used.</p>